kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/adapter/DetailedConversationAdapter.java

```diff
@@ -280,11 +291,21 @@ protected Bitmap processBitmap(Object data) {
             View v6 = layoutInflater.inflate(R.layout.km_feedback_agent_layout, parent, false);
             return new MyViewHolder6(v6);
         } else if (viewType == 0) {
-            View v0 = layoutInflater.inflate(R.layout.mobicom_received_message_list_view, parent, false);
+            View v0;
+            if(useInnerTimeStampDesign) {
+                v0 = layoutInflater.inflate(R.layout.mobicom_received_message_list_view, parent, false);
```

Comment:
@amntoppo rename variable to something meaningful, like recerverView etc

Reply:
@adarshmishra I have changed name to view0, view 1. Also, we have 6 different view holders so I have commented out the role of each of them.